### PR TITLE
[IMP] Add color to hr_skill

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Review by bodedra — APPROVED** [review]
Code review only :+1:

**Review by nikul-serpentcs — APPROVED** [review]
Looks Good :+1:

File: hr_skill/models/hr_skill.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ class Skill(models.Model):
                                          'skill_id',
                                          string='Employees')
     description = fields.Char(string="Description")
+    color = fields.Integer('Color Index', default=10)
```
Comment (tarteo) [inline hr_skill/models/hr_skill.py:25]: Why is color an integer?

Comment (max3903) [reply hr_skill/models/hr_skill.py:25]: That's how Odoo sets it on for tags and display the color selector.

**Comment by alexey-pelykh** [comment]
How this feature works?

**Comment by max3903** [comment]
@alexey-pelykh Here is an example:
https://github.com/OCA/field-service/blob/11.0/fieldservice_skill/views/fsm_order.xml#L12

Skills are displayed with the many2many_tags widget and the color allows to easily identify them.

**Comment by alexey-pelykh** [comment]
Gotcha, that explains!

File: hr_skill/models/hr_skill.py (hunk 1 above)
Comment (alexey-pelykh) [reply hr_skill/models/hr_skill.py:25]: @tarteo There are similar code pieces all over Odoo codebase

Comment (tarteo) [reply hr_skill/models/hr_skill.py:25]: Didn't know that's how it works for tags. Thanks! :+1:

**Review by tarteo — APPROVED** [review]

**Review by alexey-pelykh — APPROVED** [review]
Code LGTM

**Comment by alexey-pelykh** [comment]
Ported to v12 in #496

**Pull request merged**